packages/theme/pages/Checkout/Billing.vue

```diff
@@ -121,7 +121,7 @@
         >
           <SfSelect
             v-e2e="'billing-country'"
-            :value="billingDetails.country"
+            v-model="billingDetails.country"
```

Comment:
Could you please create an appriopriate changelog entry similar to other features we had? :)

Reply:
Done